chain/indexer/src/block.rs

```diff
@@ -97,7 +97,7 @@ where
         let db = self.db.clone();
         let tx_significant_events = self.egress.clone();
 
-        let (db_latest_block, checksum) = self.db.get_last_indexed_block(None).await?;
+        let (db_latest_block, checksum, _) = self.db.get_last_indexed_block(None).await?;
```

Comment:
At this point it'd be cleaner to replace the tuple with a dedicated struct with public members.